http.kafka.sasl.scram/stack.yml

```diff
@@ -7,14 +7,18 @@ networks:
 services:
 
   zilla:
-    image: "ghcr.io/aklivity/zilla:latest"
+    image: "ghcr.io/aklivity/zilla:develop-SNAPSHOT"
```

Comment:
I realize you need this change locally to test before the other PR is merged, but we don't actually want it in the PR, agreed?